### PR TITLE
Update zh_CN.ts

### PR DESCRIPTION
**Pull request by jingrongx**
Modify set http proxy workaround in deepin os.

### CODE REVIEW

**Comment by U-v-U** [comment]
Thanks for your contribution firstly.
This PR could not be merged due to the following problems:

1. The translation change should be pushed to version-v* branch
2. The source string of the translation *.ts should not be changed.
3. It's not Qv2ray's problem, you may consider pushing it to the website: The `Qv2ray/qv2ray.github.io` repo.
4. It's even better if the upstream Deepin workgroup fixes the bug, we'll remove the warning immediately once they released the version of DDE component with bug fixes.

**Comment by DuckSoft** [comment]
Cannot merge this. 
Reason: https://github.com/linuxdeepin/go-lib/issues/15#issuecomment-683600417

File: translations/zh_CN.ts [hunk 1]
```diff
@@ -2161,8 +2161,8 @@ Output: <byte value="xd"/>
         <translation>检测到 Deepin</translation>
     </message>
     <message>
-        <source>Deepin plays smart and sets you the wrong HTTPS_PROXY, FTP_PROXY environment variable.</source>
-        <translation>Deepin可能自作聪明，为你设置了错误的 HTTPS_PROXY 和 FTP_PROXY 环境变量。</translation>
+        <source>Deepin may sets you the wrong HTTPS_PROXY, FTP_PROXY environment variable.You can add proxy info to /home/<username>/.bashrc, such as: export https_proxy="http://127.0.0.1:8080".</source>
```
Comment (DuckSoft) [inline translations/zh_CN.ts:2164]: You can see there's some grammar error in your modification.

> Deepin may **sets** you

Shall it be:

> Deepin may have set you?

Comment (DuckSoft) [reply translations/zh_CN.ts:2164]: Also, you are not using the correct English / Chinese-English paragraph style. Multiple spaces are missing.

**Review by DuckSoft — COMMENTED** [review]
What's most, the modification won't take effect, since source strings are generated from source code. 

This pull request will simply disable the translation, with actually no effect.

Please reconsider this.

**Pull request closed without merging**